src/main/java/model/Cars.java

```diff
@@ -0,0 +1,35 @@
+package model;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cars {
+
+    private final List<Car> cars;
+
+    public Cars(List<String> carNames) {
+        this.cars = carNames.stream()
+                .map(Car::new)
+                .collect(Collectors.collectingAndThen(Collectors.toList(), List::copyOf));
+    }

```

Comment:
이전에 남겨주신 코멘트에 대한 답으로도 남겨드렸지만, `toList()`를 사용해도 무방합니다!
```suggestion
    public Cars(List<String> carNames) {
        this.cars = carNames.stream()
                .map(Car::new)
                .toList();
    }
```